production-vms/resources.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"production-vms/config"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
+)
+
+func createProductionVM(ctx context.Context, vhd VHD, subnetID string) error {
+	fmt.Printf("Creating VM %s in resource group %s\n", vhd.name, config.ResourceGroupName)
+
+	fmt.Printf("vhd resource: %s\n", vhd)
+
+	vmSize := "Standard_D8ds_v5"
+	if vhd.ImageArch == "Arm64" {
+		vmSize = "Standard_D8pds_v5"
+	}
+
+	// create unique NIC for VM
+	nicID, err := createNetworkInterface(ctx, vhd.name+"-nic", subnetID)
+	if err != nil {
+		return fmt.Errorf("cannot create NIC: %v", err)
+	}
+
+	vmParameters := armcompute.VirtualMachine{
+		Location: to.Ptr(config.Config.Location),
+		Tags: map[string]*string{
+			"SkipLinuxAzSecPack": to.Ptr("false"),
+		},
```

Comment:
@djsly is this the Tag that is required? Can confirm it shows up on the VM but I haven't looked into if AzSecPack installed

Reply:
yes